test/lint/commit-script-check.sh

```diff
@@ -17,6 +17,11 @@ if test -z "$1"; then
     exit 1
 fi
 
+if ! sed --help | grep -q 'GNU'; then
```

Comment:
```suggestion
if ! sed --help 2>&1 | grep -q 'GNU'; then
```

Could redirect stderr to stdout in order to not show any output? 

On OpenBSD's sed, there is no `--help` parameter, i.e. an error message and the usage is printed on stderr:
```
$ ./test/lint/commit-script-check.sh HEAD~100
sed: unknown option -- -
usage: sed [-aEnru] [-i[extension]] command [file ...]
       sed [-aEnru] [-e command] [-f command_file] [-i[extension]] [file ...]
Error: the installed sed package is not compatible. Please make sure you have GNU sed installed in your system.
```

Reply:
Hello @theStack,
Thank you for the suggestion, it was applied.